clloader/loader.py

```diff
@@ -101,6 +36,8 @@ def __init__(
         evaluate_on="seen",
         class_order=None
     ) -> None:
+        print("INIT!!!!!!!!!!!!!!!!!!!!!")
```

Comment:
Ahah, does it need to be kept? :p

Reply:
hahaha, je l'ai viré mais j'ai du oublier de push